src/Twig/Extension/CanonicalizeExtension.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Form\Twig\Extension;
+
+use Sonata\Form\Twig\CanonicalizeRuntime;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class CanonicalizeExtension extends AbstractExtension
+{
+    /**
+     * @return TwigFunction[]
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('sonata_form_canonicalize_locale_for_moment', [CanonicalizeRuntime::class, 'getCanonicalizedLocaleForMoment']),
```

Comment:
I didn't call it the same to avoid collisions. The SonataAdmin one should be deprecated.